cnf-app-mac-operator/Makefile

```diff
@@ -132,6 +132,10 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | ${CLUSTER_CLI} delete -f -
 
+# Include dependencies
+dependencies:
+	cp ../utils/webserver.go utils/webserver.go
```

Comment:
Here you satisfy the dependency on webserver by copying it from outside the `cnf-app-mac-operator` dir